db/models/models.js

```diff
@@ -35,3 +41,17 @@ exports.locateUsers = async () => {
   const result = await db.query(query);
   return result.rows;
 };
+
+exports.locateArticles = async () => {
+  const query = db.query(`SELECT * FROM articles`);
+  const result = await query;
+  const queryComments = db.query(
+    `select * from comments where author = '${result.rows[0].author}';`
+  );
+  const newResult = await queryComments;
+  newResult.rows.forEach((rows) => {
```

Comment:
Same point about `comment_count` here - this should be all one query please! For one thing, you don't want to just be getting the comments by a specific author, as is happening here on line 49. This can all be amended by doing it in a single query 🙂